pcsx2/Pcsx2Config.cpp

```diff
@@ -210,8 +210,8 @@ Pcsx2Config::GSOptions::GSOptions()
 	FramesToSkip			= 2;
 
 	LimitScalar				= 1.0;
-	FramerateNTSC			= 59.94;
-	FrameratePAL			= 50.0;
+	FramerateNTSC			= 59.29;
+	FrameratePAL			= 49.76;
```

Comment:
This one might be right though, for PSX only. The change would touch / break PS2 mode, as do others in this PR currently.

Reply:
I'm not sure why these values were changed. Explanation about the new values would be nice.

Reply:
Something to keep in mind is that NTSC and PAL rates are *standards* that *directly interface with external devices*, e.g. TVs and monitors. Changing these doesn't really steer into a timing system that is correct and precise, but more *bends* the current implementation into a state where it *works* but isn't necessarily the most precise, correct or efficient. At least that's how I see it.